Client/Ecosia/UI/EcosiaFindInPageBar.swift

```diff
@@ -0,0 +1,238 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0
+
+import UIKit
+import Shared
+
+protocol EcosiaFindInPageBarDelegate: AnyObject {
+    func findInPage(_ findInPage: EcosiaFindInPageBar, didTextChange text: String)
+    func findInPage(_ findInPage: EcosiaFindInPageBar, didFindPreviousWithText text: String)
+    func findInPage(_ findInPage: EcosiaFindInPageBar, didFindNextWithText text: String)
+    func findInPageDidPressClose(_ findInPage: EcosiaFindInPageBar)
+}
+
+/// Ecosia's custom UI for FindInPageBar.
+///
+/// You can find the Firefox original view in Client/Frontend/Browser/FindInPageBar (removed from Target since no longer used)
+class EcosiaFindInPageBar: UIView {
+    
+    private lazy var searchView: UIView = {
+        let view = UIView()
+        view.layer.cornerRadius = (EcosiaFindInPageBar.barHeight - 2*EcosiaFindInPageBar.searchViewTopBottomSpacing)/2
+        return view
+    }()
+    private lazy var searchTextField: UITextField = {
+        let textField = UITextField()
+        textField.addTarget(self, action: #selector(didTextChange), for: .editingChanged)
+        textField.font = .preferredFont(forTextStyle: .body)
+        textField.placeholder = .localized(.findInPage)
+        textField.autocapitalizationType = .none
+        textField.autocorrectionType = .no
+        textField.inputAssistantItem.leadingBarButtonGroups = []
+        textField.inputAssistantItem.trailingBarButtonGroups = []
+        textField.enablesReturnKeyAutomatically = true
+        textField.returnKeyType = .search
+        textField.accessibilityIdentifier = "FindInPage.searchField"
+        textField.delegate = self
+        return textField
+    }()
+    private lazy var matchCountLabel: UILabel = {
+        let label = UILabel()
+        label.font = .preferredFont(forTextStyle: .footnote)
+        label.textAlignment = .right
+        label.isHidden = true
+        label.accessibilityIdentifier = "FindInPage.matchCount"
+        return label
+    }()
+    private lazy var previousButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: "find_previous")?.withRenderingMode(.alwaysTemplate), for: .normal)
```

Comment:
How about utilising this `UIImage.templateImageNamed` handy extension? Same for all the below images 👇

Reply:
Nice, did not notice that exists, will use it